iSeg/MainWindow.cpp

```diff
@@ -59,7 +60,7 @@
 #include <QDesktopWidget>
 #include <QSignalMapper.h>
 #include <q3accel.h>
-#include <q3filedialog.h>
+#include <QFileDialog>
 #include <q3popupmenu.h>
```

Comment:
I think this is now QMenu

Reply:
maybe the next min-refactoring will replace all Q3Action and Q3PopupMenu instances. Then it will only be layout stuff left still depending on Qt3